### PR TITLE
Add Ingredient and IngredientList class

### PR DESCRIPTION
**Pull request by charkty**
For v1.0

### CODE REVIEW

File: src/main/java/essenmakanan/ingredient/Ingredient.java [hunk 1]
```diff
@@ -1,4 +1,28 @@
 package essenmakanan.ingredient;
 
 public class Ingredient {
+    private String name;
+    private String qty;
```
Comment (kaijie0102) [inline src/main/java/essenmakanan/ingredient/Ingredient.java:5]: maybe spelling out in full would be clearer? some might misinterpret as "quality"

Comment (kaijie0102) [inline src/main/java/essenmakanan/ingredient/Ingredient.java:5]: should quantity be int instead?

File: src/main/java/essenmakanan/ingredient/IngredientList.java [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
 public class IngredientList {
+    private ArrayList<Ingredient> ingredients;
+
+    public IngredientList() {
+        ingredients = new ArrayList<>();
+    }
+
+    public void addIngredient(String ingredientName) {
+        ingredients.add(new Ingredient(ingredientName));
+    }
```
Comment (kaijie0102) [inline src/main/java/essenmakanan/ingredient/IngredientList.java:13]: should parameter be Ingredient object instead?

**Review by kaijie0102 — CHANGES_REQUESTED** [review]

File: src/main/java/essenmakanan/ingredient/Ingredient.java (hunk 1 above)
Comment (charkty) [reply src/main/java/essenmakanan/ingredient/Ingredient.java:5]: good point, will change the name

File: src/main/java/essenmakanan/ingredient/IngredientList.java (hunk 2 above)
Comment (charkty) [reply src/main/java/essenmakanan/ingredient/IngredientList.java:13]: yes, will change it

**Review by kaijie0102 — APPROVED** [review]

**Pull request merged**